src/index/LocalVocabEntry.h

```diff
@@ -51,6 +56,20 @@ class alignas(16) LocalVocabEntry
   QL_EXPLICIT(false) LocalVocabEntry(const Base& base) : Base{base} {}
   QL_EXPLICIT(false)
   LocalVocabEntry(Base&& base) noexcept : Base{std::move(base)} {}
+  // Constructor for when the position in the vocab is already known.
+  QL_EXPLICIT(true)
+  LocalVocabEntry(Base&& base, auto lower, auto upper)
+      : Base{std::move(base)},
+        lowerBoundInVocab_(IdProxy::make(lower.getBits())),
+        upperBoundInVocab_(IdProxy::make(upper.getBits())),
+        positionInVocabKnown_(true) {
+    // Check that the given bounds are correct. The extra braces are needed to
+    // keep the macro expansion from interpreting the expression as two separate
+    // parameters.
+    AD_EXPENSIVE_CHECK((positionInVocabExpensiveCase() ==
+                        PositionInVocab{IdProxy::make(lower.getBits()),
+                                        IdProxy::make(upper.getBits())}));
```

Comment:
Entweder die Änderung in `positionInVocabExpensive` rückgängig machen oder hier die `PositionInVocab` mit den Membern erstellen